### PR TITLE
fix-dev-docs-link

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

fix link from [dev](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md) file to docs

# Code review checklist

- [X] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [X] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [X] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:mrjones-fix-dev-docs-link/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:mrjones-fix-dev-docs-link/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:mrjones-fix-dev-docs-link/docker-compose/cht-couchdb-clustered.yml)
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by andrablaj — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Development Setup (DEPRECATED)
 
-This content has been moved to [our docs site](https://docs.communityhealthtoolkit.org/apps/guides/hosting/core-developer/).  
+This content has been moved to [our docs site]([https://docs.communityhealthtoolkit.org/apps/guides/hosting/core-developer](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/)).
```
Comment (yrimal) [inline DEVELOPMENT.md:3]:
```suggestion
This content has been moved to [our docs site](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/). 
```

**Review by yrimal — CHANGES_REQUESTED** [review]
The link appears to be still broken, I've suggested the correct link inline.

**Comment by andrablaj** [comment]
Thanks @yrimal for catching this! I checked the new URL without running cht-docs locally, my bad. 🤦‍♀️

**Comment by mrjones-plip** [comment]
Yes - Good - catch - thank you @yrimal !

**Pull request merged**